cpp/locality/Voronoi.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2010-2019 The Regents of the University of Michigan
+// This file is from the freud project, released under the BSD 3-Clause License.
+
+#ifndef VORONOI_H
+#define VORONOI_H
+
+#include "VectorMath.h"
+#include <vector>
+
+using namespace std;
+
+namespace freud { namespace locality {
+
+struct NeighborBond {
+    NeighborBond() : index_i(0), index_j(0), weight(0) {}
+
+    NeighborBond(unsigned int index_i, unsigned int index_j, float w) :
+        index_i(index_i), index_j(index_j), weight(w) {}
+
+    //! Equality checks both i, j pair and weight.
+    bool operator== (const NeighborBond &n)
+        {
+        return (index_i == n.index_i) && (index_j == n.index_j) && (weight == n.weight);
+        }
+
+    //! Default comparator of points is by weight.
+    /*! This form of comparison allows easy sorting of nearest neighbors by
+     *  weight.
+     */
+    bool operator< (const NeighborBond &n) const
+    {
+        if (index_i < n.index_i) {
+            return true;
+        } else if (index_j < n.index_j) {
+            return true;
+        }
+        return weight < n.weight;
+    }
```

Comment:
Actually the current implementation doesn't need this comparing function. I include this because I'm not sure if it will be needed in the future. Would you still like me to change per your suggestions?

Reply:
@yzjin I would remove this logic if it's not being used, just to prevent confusion.

Reply:
Updated